satellite/metainfo/metainfo.go

```diff
@@ -1840,6 +1878,7 @@ func (endpoint *Endpoint) DownloadSegment(ctx context.Context, req *pb.SegmentDo
 			}
 			segmentMeta = streamMeta.LastSegmentMeta
 		} else {
+			segmentMeta = &pb.SegmentMeta{}
```

Comment:
why are we adding this?

Reply:
otherwise, its `nil` and unmarshaling will fail